runtime/doc/vim9.txt

```diff
@@ -1757,8 +1757,7 @@ be inferred, then the type is "any": >
 Declaring a function, see |vim9-func-declaration|, is particularly unique.
 
 						*null-compare*
-For familiar null compare semantics, where a null container is not equal to
-an empty container, do not use null_<type> in a comparison: >
+It maybe a bit confusing to use null or null_<type> in a comparison: >
```

Comment:
The programmer must decide which one to pick. Why do you take out the reasoning for which one to pick: `null` versus `null_<type>`? The question is: does the programmer want to distinguish null container from empty container. The difficulty in doing that is one of the main points of discussion in #13458.

Reply:
yes, it is their choice, no `do not use` such thing here.